protobuf/protoc-gen-cruxclient/generated/routeguide/v1/message.djinni.jni.h

```diff
@@ -209,10 +209,10 @@ struct Translator {
 #include "routeguide/v1/message.pb.h"
 namespace djinni::routeguide::v1::RouteSummary::Details {
 struct JNIInfo {
-  const GlobalRef<jclass> clazz { jniFindClass("io/grpc/examples/routeguide/RouteSummary_Details")  };
+  const GlobalRef<jclass> clazz { jniFindClass("io/grpc/examples/routeguide/RouteSummary$Details")  };
   const jmethodID method_toBytes { jniGetMethodID(clazz.get(), "toByteArray", "()[B") };
   const jmethodID method_byteSize { jniGetMethodID(clazz.get(), "getSerializedSize", "()I") };
-  const jmethodID method_fromBytes { jniGetStaticMethodID(clazz.get(), "parseFrom", "([B)Lio/grpc/examples/routeguide/RouteSummary_Details;") };
+  const jmethodID method_fromBytes { jniGetStaticMethodID(clazz.get(), "parseFrom", "([B)Lio/grpc/examples/routeguide/RouteSummary$Details;") };
```

Comment:
@tmoura10  does this change to generated code looks good to you?

Reply:
Yeah, that's what I did on my tests, changing both `jniFindClass` and `jniGetStaticMethodID`